wormchain/app/app.go

```diff
@@ -420,9 +440,13 @@ func New(
 
 	// this line is used by starport scaffolding # stargate/app/keeperDefinition
 
+	app.ContractKeeper = wasmkeeper.NewDefaultPermissionKeeper(app.wasmKeeper)
+	app.Ics20WasmHooks.ContractKeeper = app.ContractKeeper
+	app.WormholeMwKeeper.SetWasmKeeper(&app.wasmKeeper)
```

Comment:
Why can't we initialize all the middlewares after we initialize the wasmkeeper?
It looks like it might be because we edit the transferKeeper in `app.WireICS20PreWasmKeeper`? But if the transferKeeper is a pointer, we should be able to remove this?

Reply:
The transferKeeper, which needs the middlewares setup, needs to be created before the wasmKeeper is created and uses it.